### DISCUSSION TITLE
Change in Quarto from Rstudio RStudio 2022.07.1+554 to 2022.07.2+576

### DISCUSSION THREAD

**Discussion by srowelluf** [quarto-dev/quarto-cli#3357] [Q&A]
Do you know if there is a change in how quarto should render HTML from the version shipped with Rstudio 2022.07.1+554 to 2022.07.2+576 ?

I had updated to Rstudio 2022.07.2+576 and a document that I render daily to HTML suddenly had slightly larger font size, and a gt table that always fit on the screen suddenly needed a scroll bar. I then reverted back to 2022.07.1+554 and re-rendered and I was back to normal.

I can create a dummy quarto doc to upload - but first I thought I would check if there was a known change that would account for it. Looking at the release notes I see "Support for v2 format of Quarto crossref index" but didn't know what that meant.

**Comment by mcanouil** [comment]
This change or at least Quart version embedded in both RStudio versions should be stated in the changelog.
Or you can check Quarto's version between the two then refer to Quarto changelog.
Note that Quarto changelog does not focus on "patch", but on "minor".

I think the older version embed v1.0 and the newest version embed v1.1.
FYI, v1.2 was released earlier this week.

**Reply by dragonstyle** [reply]
@mcanouil is correct- the early version included Quarto 1.0 and the later version included Quarto 1.1, which had a pretty substantial number of changes. I'm not aware of changes purposely creating the format changes you describe, but it seems likely that something changed that is causing this.

**Reply by srowelluf** [reply]
This does seem to be due to change in quarto version. If I change to a newer version of quarto (including 1.2.269) in RStudio 22.07.1+554; I get the same results as I do in RStudio 2022.7.2+576

Inline everything looks as expected - but different look when knitting to HTML

Here is an example of a difference in the same code and what it looks like in Quarto 1.0.36 vs. Quarto 1.1.189 OR 1.2.269 (knitr 1.41)

`---
title: "Testing tables"
format: 
  html
editor: visual
---


```{r}
library(tidyverse)
library(gt)
library(gtExtras)
library(glue)
```


## GT 0.8.0

cylinders 

```{r}
mtcars %>% 
  slice_max(order_by = mpg, n = 5) %>% 
  gt() %>% 
  cols_label(mpg = "miles per gal",
             cyl = "Number of cylinders",
             disp = "Displacement (cu.in.)",
             hp = "Gross horsepower",
             drat = "Rear axle ratio",
             wt = "Weight (lb/1000)",
             qsec = "1/4 mile time",
             vs = "V/S",
             am = "Transmission (0 = automatic, 1 = manual)"
             )
  
```
`
Quarto 1.1.189 OR 1.2.269
![image](https://user-images.githubusercontent.com/32438999/202750943-9084f941-c799-461c-ad56-f377d7736b75.png)

Quarto 1.0.36

![image](https://user-images.githubusercontent.com/32438999/202751408-1233d5e9-267b-4f10-b9ac-9f2b97576e6e.png)

 note the view from 1.0.36 matches the inline output when run interactively.  The Quarto 1.1.189 OR 1.2.269 versions do not match the interactive view

**Reply by mcanouil** [reply]
Probably caused by some CSS overwriting the table CSS.
If you don't want Quarto or anything actually to take over the styling, then you should consider to use `gt::as_raw_html(data, inline_css = TRUE)` (https://gt.rstudio.com/reference/as_raw_html.html).

**Reply by srowelluf** [reply]
Thank you @mcanouil  that fixed my issue. 

There are other places in the body that look slightly different in 1.1.189 and 1.2.269; but not really problematic - this one was causing me grief.

**Reply by srowelluf** [reply]
looking through closed issues; I'm thinking this may be related to: https://github.com/quarto-dev/quarto-cli/pull/3039
